.codex/hooks/log_stop.py

```
@@ -0,0 +1,60 @@
+            "ts": utc_now(),
+            "event": payload.get("hook_event_name", "Stop"),
+            "session_id": payload.get("session_id"),
+            "turn_id": payload.get("turn_id"),
+            "cwd": cwd,
+            "model": payload.get("model"),
+            "stop_hook_active": payload.get("stop_hook_active"),
+            "last_assistant_message_length": len(last_message) if isinstance(last_message, str) else None,
+        }
+        append_jsonl(root, "turns.jsonl", record)
+    except Exception:
+        pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)